scripts/docker/requirements-20.04.txt

```diff
@@ -8,7 +8,7 @@ pandoc
 libtbb-dev
 libcapstone-dev
 pkg-config
-cmake=3.16.3-1ubuntu1.20.04.1
+cmake
```

Comment:
should we just pin to a later version for this?

Reply:
What's the advantage of holding back to an old(er) version?
Normally it's a fear that a newer version might break things.  But the CMake authors bend over backwards to maintain backwards compatibility, and items that do eventually get deprecated sit around for a couple years' worth of releases before they're finally dropped.

IMO it's more work for us perpetually bumping that pinned version than to reactively pin to an older version in the unlikely event that we need to.  My 2c.

Reply:
After discussion offline, we've decided to pin to Ubuntu 22.04's default version (v3.22).